tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
+    "target": "es2015",
+    "moduleResolution": "node",
+    "noEmit": true
```

Comment:
Just in case you didn’t know: At some point you probably want to make the typechecking more comprehensive, especially adding `"strict": true`. More candidates:

```json
    "strict": true,
    "noFallthroughCasesInSwitch": true,
    "noImplicitReturns": true,
    "useUnknownInCatchVariables": true
```

Reply:
Thanks! I knew I didn't check for much (because I first want to get rid of the configuration-related errors), but I didn't know exactly which things I needed to be enable. This saves me some times :blush: 
If you have more tips and candidates, I'll take them. I think there was one about implicit any for instance, which I'll try to find at some point.

Reply:
`strict` is a shortcut for enabling a bunch of other options, including `noImplicityAny`. I couldn’t find the exact list of options in TypeScript’s documentation, but I found this in a blog post:

```
    noImplicitAny
    noImplicitThis
    alwaysStrict
    strictBindCallApply
    strictNullChecks
    strictFunctionTypes
    strictPropertyInitialization
```

I went through the [elm-watch tsconfig.json](https://github.com/lydell/elm-watch/blob/main/tsconfig.json), and the last strictness related one seems to be:

```json
    "noUncheckedIndexedAccess": true,
```

But when adding TypeScript to an already existing code base it might make sense to enable one thing at a time so you don’t drown in errors, before you have even learned how to deal with many of them.

The final level is to bring in typscript-eslint. Then you get close to Elm level type system confidence.

Reply:
Perfect, thanks :)